html5lib/tests/tree_construction.py

```diff
@@ -51,14 +51,17 @@ def runtest(self):
         fragmentContainer = self.test['document-fragment']
         expected = self.test['document']
         expectedErrors = self.test['errors'].split("\n") if self.test['errors'] else []
+        script = True
+        if 'script-off' in self.test:
+            script = False
```

Comment:
Uh, IIRC everything under `tree-construction/scripted/` should have `#script-on`. I'd expect `tree-construction/scripted/webkit01.dat` to fail currently—ideally we'd skip it because there's no way we can pass it without real scripting support.